.github/workflows/main.yml

```diff
@@ -104,22 +104,22 @@ jobs:
         pytest
         python tests/integration/test_config.py
     - name: Test heartbeats (pwsh)
-      if: matrix.os == 'windows-latest' && (matrix.cversion == '24.11.3' || matrix.cversion == '25.1.1')

```

Comment:
Only verifying heartbeats on one of the conda versions for now while we get to the root of issues that seem related to proxyspy socket usage issues on Linux ARM